web/atlas.json

```diff
@@ -3333,7 +3333,6 @@
 {"id": "tx1yql", "submitted_by": "cuddle_curve", "name": "Vötgil", "description": "Vötgil is a constructed language made by Jack Eisenmann in 2012, and is intended to be a simpler version of English. It is spelled either using the Latin or Vötgil alphabet. Each letter of the Vötgil alphabet is a column of 3 cells, each cell being either white, light gray. dark gray, or black.nThe language was popularized in 2016 by jan Misali, and since has been a inside joke for fans of constructed languages.", "website": "http://www.ostracodfiles.com/votgil/main.html", "subreddit": "/r/V0tgil", "center": [753.5, 328.5], "path": [[750.5, 327.5], [755.5, 327.5], [755.5, 329.5], [750.5, 329.5]]},
 {"id": "tx1ybp", "submitted_by": "JessyPkLover", "name": "Gallic cockerel", "description": "The Gallic cockerel is one of the symbols of France, but is not an official symbol of the French Republic.", "website": "https://www.elysee.fr/la-presidence/le-coq", "subreddit": "/r/franceplace", "center": [1144.5, 785.5], "path": [[1135.5, 781.5], [1140.5, 781.5], [1142.5, 784.5], [1147.5, 784.5], [1146.5, 782.5], [1140.5, 777.5], [1148.5, 771.5], [1150.5, 776.5], [1152.5, 776.5], [1153.5, 777.5], [1153.5, 780.5], [1152.5, 781.5], [1151.5, 783.5], [1153.5, 786.5], [1153.5, 788.5], [1150.5, 792.5], [1150.5, 797.5], [1145.5, 798.5], [1145.5, 795.5], [1146.5, 795.5], [1146.5, 793.5], [1143.5, 791.5], [1138.5, 795.5], [1135.5, 792.5], [1132.5, 791.5], [1132.5, 789.5], [1135.5, 785.5], [1138.5, 784.5], [1137.5, 784.5], [1135.5, 782.5], [1134.5, 781.5]]},
 {"id": "tx1y8s", "submitted_by": "slatourelle", "name": "Houdini", "description": "Houdini is a leading VFX and 3d software used for making VFX and tools for the games and film industries", "website": "https://www.sidefx.com/", "subreddit": "/r/houdini", "center": [1478.5, 1653.5], "path": [[1483.5, 1658.5], [1483.5, 1647.5], [1472.5, 1647.5], [1472.5, 1658.5]]},
-{"id": "tx1y4k", "submitted_by": "MineWarz", "name": "The Button", "description": "A reference to the popular 2015 Reddit April Fools' event 'The Button'. Made by the April Knights.", "website": "", "subreddit": "/r/thebutton", "center": [809.5, 672.5], "path": [[804.5, 670.5], [814.5, 670.5], [814.5, 673.5], [804.5, 673.5]]},

```

Comment:
Destroyed art is allowed. We are working on the ability to time-box them so they only appear at the proper points on the timeline.